### PR TITLE
Fix BBB to run on latest `master`

### LINKED ISSUE #69: Don't use MutableArtifacts.toml anymore, use Scratch
MutableArtifacts were always a kludge due to the lack of scratch spaces.  We should write out UID-remapped squashfs images to scratch spaces (namespaced appropriately to avoid interfering with other squashfs images).

### PR DESCRIPTION
**Pull request by staticfloat**
This contains some fixes to BBB to support running on Julia v1.6+.  It requires a Julia that contains a Pkg with this PR: https://github.com/JuliaLang/Pkg.jl/pull/2318

Fixes #69.

### CODE REVIEW

**Comment by giordano** [comment]
> It requires a Julia that contains a Pkg with this PR: JuliaLang/Pkg.jl#2318

When that finds its way into a nightly build, you need to revert https://github.com/JuliaPackaging/BinaryBuilderBase.jl/commit/6da0fa5a1af79c6dade25d39e962711949d9248d

**Review by giordano — COMMENTED** [review]
I had these very important comments pending for 12 days

File: Manifest.toml [hunk 1]
```diff
@@ -27,11 +27,6 @@ uuid = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 deps = ["Markdown"]
 uuid = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 
-[[JLLWrappers]]
-git-tree-sha1 = "7cec881362e5b4e367ff0279dd99a06526d51a55"
-uuid = "692b3bcd-3c85-4b1f-b108-f13ce0eb3210"
-version = "1.1.2"
-
```
[the remark this answers is not in the document]
Comment (staticfloat) [reply Manifest.toml:34]: .....huh.  Why did this happen?

[the remark this answers is not in the document]
Comment (staticfloat) [reply Manifest.toml:34]: Ahhh, it's because all the JLLs here are stdlibs!  Hilarious.

**Comment by staticfloat** [comment]
We now need to wait for the `backports-release-1.6` branch to be merged, then we'll be good to go.

**Comment by giordano** [comment]
It's all green! :open_mouth:

**Pull request merged**